extensions/amp-carousel/0.1/slides.js

```diff
@@ -87,7 +87,14 @@ export class AmpSlides extends BaseCarousel {
   }
 
   /** @override */
-  goCallback(dir, animate) {
+  goCallback(dir, animate, opt_autoplay) {
+    // if `opt_autoplay` is not set, this means a user interacted with the
+    // the carousel and we cancel any current and future autoplay request
+    if (!opt_autoplay) {
```

Comment:
I don't understand why this is needed. If the user starts a swipe, we'll cancel the the auto-animate timeout. Further, this is always bound with `true`, so how would it be false later except after the swipe started (and we've already canceled)?